corehq/blobs/migrate.py

```diff
@@ -415,8 +415,12 @@ def get_doc_migrator(self, filename, date_range=None, **kw):
         migrator = super(BackendMigrator, self).get_doc_migrator(filename)
         return _migrator_with_worker_pool(migrator, self.reindexer, **kw)
 
-    def get_document_provider(self):
-        return SqlDocumentProvider(self.iteration_key, self.reindexer)
+    def get_document_provider(self, kw):
```

Comment:
can you mane this `get_document_provider(self, date_range=None, **kw)` so that it's clear it takes a `date_range` kwarg.

Reply:
Updated.